source/Hardpoint.cpp

```diff
@@ -227,10 +230,34 @@ void Hardpoint::Fire(Ship &ship, vector<Projectile> &projectiles, vector<Visual>
 	// to share the same inaccuracy as the projectile.
 	aim += Projectile::Inaccuracy(outfit->Inaccuracy());
 
-	// Create a new projectile, originating from this hardpoint.
-	// In order to get projectiles to start at the right position they are drawn
-	// at an offset of (.5 * velocity). See BatchDrawList.cpp for more details.
-	projectiles.emplace_back(ship, start - .5 * ship.Velocity(), aim, outfit);
+	if(outfit->ShipToShoot().empty())
+		// Create a new projectile, originating from this hardpoint.
+		// In order to get projectiles to start at the right position they are drawn
+		// at an offset of (.5 * velocity). See BatchDrawList.cpp for more details.
+		projectiles.emplace_back(ship, start - .5 * ship.Velocity(), aim, outfit);
+	else
+	{
+		const Ship *shipToPlace = GameData::Ships().Get(outfit->ShipToShoot());
+		if(shipToPlace->IsValid())
+		{
+			const shared_ptr<Ship> shotShip = make_shared<Ship>(*shipToPlace);
+			shotShip->Recharge();
+			shotShip->SetName(ship.Name());
+			shotShip->SetGovernment(ship.GetGovernment());
+			if(outfit->ShipToShootPersonality().IsDefined())
+				shotShip->SetPersonality(outfit->ShipToShootPersonality());
+			else
+				shotShip->SetPersonality(ship.GetPersonality());
+			shotShip->SetHail(*ship.GetHailPhrase());
+			shotShip->SetSystem(ship.GetSystem());
+			if(ship.GetParent() && !shotShip->GetPersonality().IsCoward())
+				shotShip->SetParent(ship.GetParent());
+			shotShip->Place(start, ship.Velocity() + aim.Unit() * outfit->Velocity(), aim);
+			newShips.push_back(shotShip);
+			if(ship.IsYours())
+				player.AddShip(shotShip);
```

Comment:
Is this why you made this part of the gun and not the submunition?
Honestly I would prefer if the auto fired ships were not added to your ships (I'm not going to pick up after them), or at least it would be neat to be able to define a government for them.

Reply:
hrmmmm.... So what would you do to have them fight for the player?

Reply:
Could be done in amother PR but mb default is this and if you define a gov its whatever gov you define it as without being added to the list?

Reply:
hmmm, will try some things out

Reply:
You can probably give the ship the player government without adding it to the player's ship list.

Reply:
Hmmm, probably yes, if thats what we want?

Reply:
idk

Reply:
maybe we need some more opinions, putting this on a wait for now